main/utils/nameList.c

```diff
@@ -0,0 +1,296 @@
+/**
+ * @file nameList.c
+ * @author Jeremy Stintzcum (jeremy.stintzcum@gmail.com)
+ * @brief Provides index-based usernames that will avoid the issues inherent in free text entry.
+ * @date 2025-05-02
+ *
+ * @copyright Copyright (c) 2025
+ *
+ */
+
+//=============================================================================
+// Includes
+//==============================================================================
+
+// Swadge
+#include "nameList.h"
+#include "macros.h"
+
+// C
+#include <string.h>
+
+// ESP
+#include <esp_wifi.h>
+#include <esp_log.h>
+#include <esp_random.h>
+
+//=============================================================================
+// Defines
+//==============================================================================
+
+#define USER_LIST_SHIFT 1
+
+//==============================================================================
+// Consts
+//==============================================================================
+
+static const char* const adjList1[] = {
```

Comment:
May want to run words by #super-accessibility to see if anyone would get offended.